data/json/items/battery.json

```diff
@@ -4,306 +4,143 @@
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 2 } } ]
   },
   {
     "id": "light_minus_disposable_cell",
+    "copy-from": "light_minus_battery_cell",
+    "//": "cr2032, Lithium-Manganese, duracell CR2032 but they all pretty equal in their range",
     "type": "MAGAZINE",
-    "category": "tool_magazine",
     "name": { "str": "ultra-light disposable battery", "str_pl": "ultra-light disposable batteries" },
-    "description": "A light battery cell designed for small size over everything else.  It retains its universal compatibility, though.  The battery's chemistry means that it has a very high capacity, but cannot be recharged.",
+    "description": "A light LMO battery cell designed for small size over everything else.  Used for watches, calculators, toys or similar low-power electronics.",
     "ascii_picture": "light_minus_disposable_cell",
-    "weight": "3 g",
-    "volume": "1 ml",
-    "price": "30 USD",
-    "price_postapoc": "1 USD",
-    "material": [ "iron", "plastic" ],
-    "symbol": "=",
-    "color": "yellow",
-    "ammo_type": [ "battery" ],
-    "count": 2,
-    "capacity": 2,
-    "looks_like": "battery",
-    "flags": [ "NO_SALVAGE", "NO_UNLOAD", "NO_RELOAD", "BATTERY_ULTRA_LIGHT" ],
+    "delete": { "flags": [ "RECHARGE" ] },
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 2 } } ]
   },
   {
     "id": "light_battery_cell",
     "type": "MAGAZINE",
     "category": "tool_magazine",
-    "name": { "str": "light battery", "str_pl": "light batteries" },
-    "description": "A light battery cell, universally compatible with all kinds of small devices.",
-    "ascii_picture": "light_battery_cell",
-    "weight": "30 g",
-    "volume": "25 ml",
-    "price": "35 USD",
-    "price_postapoc": "1 USD",
-    "material": [ "iron", "plastic" ],
-    "symbol": "=",
-    "color": "yellow",
-    "ammo_type": [ "battery" ],
-    "capacity": 22,
-    "looks_like": "battery",
-    "flags": [ "NO_SALVAGE", "NO_UNLOAD", "NO_RELOAD", "RECHARGE", "BATTERY_LIGHT" ],
-    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 22 } } ]
-  },
-  {
-    "id": "light_plus_battery_cell",
-    "type": "MAGAZINE",
-    "category": "tool_magazine",
     "name": { "str": "light battery (high-capacity)", "str_pl": "light batteries (high-capacity)" },
-    "description": "A high-capacity light battery cell, universally compatible with all kinds of small devices.",
-    "ascii_picture": "light_plus_battery_cell",
-    "weight": "45 g",
-    "volume": "35 ml",
-    "price": "50 USD",
-    "price_postapoc": "1 USD 50 cent",
-    "material": [ "iron", "plastic" ],
+    "description": "A light, high-capacity AA battery cell, popular for any consumer-grade electronic: from remotes to mouses, from clocks to thermometers.",
+    "//": "AA, Li-Ion, ENERGIZER L91",
```

Comment:
[L91](https://data.energizer.com/pdfs/lithiuml91l92_appman.pdf) are Lithium/Iron Disulfide , not Li-ion